code/modules/mining/fulton.dm

```diff
@@ -78,6 +78,7 @@ GLOBAL_LIST_EMPTY(total_extraction_beacons)
 	if(!isturf(thing.loc)) // no extracting stuff inside other stuff
 		return
 	if(thing.anchored || (thing.move_resist > max_force_fulton))
+		to_chat(user, span_warning("[thing] is too heavy for the [src] to lift!"))
```

Comment:
Very low stakes error feedback? Smells like this should be a balloon alert to me